web_src/js/index.js

```diff
@@ -64,10 +64,16 @@ function initEditPreviewTab($form) {
     previewFileModes = $previewTab.data('preview-file-modes').split(',');
     $previewTab.on('click', function () {
       const $this = $(this);
+      let context = `{$this.data('context')}/`;
+      const treePathEl = $form.find('input#tree_path');
+      if (treePathEl) {
```

Comment:
Wouldn't this be safer?
```suggestion
      if (treePathEl.length > 0) {
```